### PR TITLE
First Capstone Project at Microverse

### PR DESCRIPTION
**Pull request by emhamza**
### These are the steps that I have followed to develop my first capstone project at Mircroverse
[National Educational Summit Loom video](https://www.loom.com/share/692f919f65144287be82b610bec2b47d)
- [x] Planning a Road map
- [x] Linters configuration done
- [x] Mobile-first model added
- [x] Dynamic design for Speakers' section
- [x] Mobile menu added
- [x] Responsive design
- [x] Site Deployment
- [x] Desktop menus added
- [x] Two Page website (mobile-first) completed

### CODE REVIEW

**Review by AlphaNtihinduka — COMMENTED** [review]
Hi @emhamza ,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

# Invalid Code Review Request ♻️ 

- I have noticed that you did not include the linters for javascript and yet you have a javascript file (`fun.js`). Kindly ensure that [Eslint](https://github.com/microverseinc/linters-config/tree/master/html-css-js#eslint) is installed to ensure that the javascript file has no linter erros ☺️ .

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready pray

Cheers and Happy coding!clapclapclap

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+            <li class="mobile-items">
+                <a href="#"  class="nav-red mx-2">CC Compaign</a>
+            </li>
+        </ul>   
+        <div class="hamburger">
+            <span class="bar"></span>
+            <span class="bar"></span>
+            <span class="bar"></span>
+        </div>
+    </nav>
```
Comment (ShadyShawkat) [inline index.html]: - [x] Kindly make sure that the navbar's high is close to or similar to the original design.

| Your design | Original design |
| --- | --- |
| image | image |

File: index.html [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+    </main>
+        <!-- The programme section goes there -->
+        <section class="program" id="events">
+            <h2 class="program-title text-center mx-3">The Conference Includes</h2>
+            <div class="br-line"></div>
+            <div class="event-container">
+                <div class="single-event">
+                    <svg xmlns="http://www.w3.org/2000/svg" width="50" height="50" fill="currentColor" class="bi bi-filetype-key x-1" viewBox="0 0 16 16">
+                        <path fill-rule="evenodd" d="M14 4.5V14a2 2 0 0 1-2 2h-1v-1h1a1 1 0 0 0 1-1V4.5h-2A1.5 1.5 0 0 1 9.5 3V1H4a1 1 0 0 0-1 1v9H2V2a2 2 0 0 1 2-2h5.5L14 4.5ZM3.21 11.85h-.87L.83 13.64H.79v-1.79H0v3.999h.791v-1.283l.41-.466 1.12 1.749h.951l-1.488-2.276 1.427-1.723Zm2.903 3.352h-1.79v-1.073h1.685v-.606H4.323v-1.025h1.79v-.648H3.538v3.999h2.575v-.647Zm2.243-.888v1.535h-.794v-1.52L6.223 11.85H7.1l.853 1.696h.032l.855-1.696h.856l-1.339 2.464Z"/>
+                    </svg>
```
Comment (ShadyShawkat) [inline index.html:100]: - [x] Please make sure that the high of the cards is similar to the original design on the desktop view.

| Your design | Original design |
| --- | --- |
| image | image |

File: index.html [hunk 3]
```diff
@@ -0,0 +1,195 @@
...
+           </div> 
+           <div>
+            <img class="partners-pic" src="./images/Additional+Needs+Web+(1).png" alt="">
+           </div> 
+           <div>
+            <img class="partners-pic" src="./images/Literacy+Masterclass.png" alt="">
+           </div> 
+           <div>
+            <img class="partners-pic" src="./images/freeseminarsfloat.png" alt="">
+           </div>
```
Comment (ShadyShawkat) [inline index.html]: - [x] The "Partners" section is too big on the desktop view. Kindly make sure that this section's size is appropriate to the original design.

| Your design | Original design |
| --- | --- |
| image | image |

File: about.html [hunk 4]
```diff
@@ -0,0 +1,155 @@
...
+        <footer class="footer">
+            <h3 class="footer-1 text-center">
+                The National<br> Educational Summit
+            </h3>
+            <div>
+                <p class="footer-2 px-3">
+                    Originally Designed by Chindy Shin Behance, <br>some rights reserved 
+                </p>
+            </div>
+        </footer>
```
Comment (ShadyShawkat) [inline about.html]: - [x] Please remove the "Partners" section from the "About" page to match the given design.
- [x] Please adjust the background color of the footer section on the "About" page.

| Your design | Original design |
| --- | --- |
| image | image |

File: readme.md [hunk 5]
```diff
@@ -0,0 +1,199 @@
...
+- **[Dynamically designed]**
+- **[DOM Manipulation]**
+- **[Scroldown menu for mobile version]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [<a href="https://emhamza.github.io/Capstone-1/" target="_blank">Online version</a>](#)
+
```
Comment (ShadyShawkat) [inline readme.md:79]: - [x] Please make sure that the online version is up and has a working link in your readme.

image

File: index.html [hunk 6]
```diff
@@ -0,0 +1,195 @@
...
+            <smal class="c-1 fs-5">
+                Brisbane
+            </small>
+        </logo>
+
+        <!-- desk main navbar -->
+        <div class="desk-main-nav">
+          <div>
+            <a href="about.html" class="text-sz mx-2">About</a>
+            <a href="#events"  class="text-sz mx-2">Events</a>
```
Comment (ShadyShawkat) [inline index.html]: - [x] Kindly remove the logo in the navbar on the mobile view and move the hamburger menu to the left side to match the given template.

| Your design | Original design |
| --- | --- |
| image | image |

File: fun.js [hunk 7]
```diff
@@ -0,0 +1,86 @@
...
+                                          <h3 class='speaker-post'>${speakerInfo[i].post}</h3>
+                                          <hr class='speaker-hr'>
+                                          <p class='speaker-data'>${speakerInfo[i].text}</p>
+                        </div>
+                    </div>
+                      
+        `;
+  }
+  personalities.innerHTML = displayPer;
+}
```
Comment (ShadyShawkat) [inline fun.js:84]: - [x] Please make sure that the elements don't overlap in the "Speakers" section on the mobile view.

image

File: index.html [hunk 8]
```diff
@@ -0,0 +1,195 @@
...
+            </div>
+        </section>
+        <!-- Speaker section goes there -->
+        <section>
+            <h2 class="text-center mt-5">Featured Speakers</h2>
+            <div class="br-line"></div>
+                <article class="speaker-x mt-5" id="speakers">
+                </article>
+
+        </section>
```
Comment (ShadyShawkat) [inline index.html]: - [x] Please add the "More" button to the end of the "Speakers" section on the mobile view to match the given template.

image

File: index.html [hunk 9]
```diff
@@ -0,0 +1,195 @@
...
+        <footer class="footer">
+            <h3 class="footer-1 text-center">
+                The National Educational <br>Summit
+            </h3>
+            <div>
+                <p class="footer-2 px-3">
+                    Originally Designed by Chindy Shin Behance,<br> some rights reserved 
+                </p>
+            </div>
+        </footer>
```
Comment (ShadyShawkat) [inline index.html]: - [x] Please remove the "Partners" and the "Footer" section from the index page on mobile view as they don't exist in the original design.

**Review by ShadyShawkat — CHANGES_REQUESTED** [review]
## Hi @emhamza, 

Good job so far! 👍🏻 
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight
- All linters are passing ✔️ 
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 

Image

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 8 above)
Comment (emhamza) [reply index.html]: Definitely, thank you so much pointing that out

File: fun.js (hunk 7 above)
Comment (emhamza) [reply fun.js:84]: Thanks for your progressive feedback, I will surely consider these changes in the mobile version

File: index.html (hunk 6 above)
Comment (emhamza) [reply index.html]: Positive! I misjudged the instructions but now I understand, thank you so much for pointing that out @ShadyShawkat

File: readme.md (hunk 5 above)
Comment (emhamza) [reply readme.md:79]: You got my bug, I have missed it earlier, now I got that and I would like to thank you for that

File: about.html (hunk 4 above)
Comment (emhamza) [reply about.html]: Definitely, This is what review for!Thanks Sir, for pointing out my mistakes

**Review by Owusu-Desmond — APPROVED** [review]
Hi @emhamza , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

 

## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**